src/app/Shared/Redux/Configurations/DashboardConfigSlice.tsx

```diff
@@ -261,6 +266,14 @@ const getTemplateHistoryIndexForMutation = (state: DashboardConfigState, templat
 
 export const dashboardConfigReducer = createReducer(INITIAL_STATE, (builder) => {
   builder
+    .addCase(dashboardConfigClearAllCardsIntent, (state) => {
+      const currentLayout = state.layouts[state.current];
+      if (currentLayout.cards.length === 0) {
```

Comment:
Does this need to be checked? If this is true then it's already an empty list, so just assigning the empty list again would be harmless.